### PR TITLE
feat(new command): override the initial commit message

### LINKED ISSUE #19: Change the initial commit of generated ng2 projects
Currently ember-cli initializes the git repo with the following commit message:

```
Author: Tomster <tomster@emberjs.com>
Date:   ..

    Initial Commit from Ember CLI v1.13.1

                                                            _...,
                                                         ,:^;,...;
                  -+===;.         ,,--++====++-,,,     .:  /.....,
                :::::~+++++#:,+#++++++++++++++++++#*..:   /,......
               (,,,,,,::=+++##++++++++++++++++++++++#.   :....../
                ...,,,,,::++++++++++++++++++++++++++++++*..,...:
                *..+...,#@@@@@@@@@++++++++++++++++++++++#*....*
                  @#,;##############@@@+*+#@@@@@@@@@@#*++#..<
                  *@##@@+,-*^^^*-+@####@@@######@@@#####@@,,,+
                    @#@*            @#@@@@#@@+--*^^*--#@@@@@@#
                    @#@.    @#      @##+++@#,           .@@#@@
                     #@#    @@     +@@++++#@@     @@     :@@
                     :@#*         @#@++++++@#*    #@     @@+
                    :*+@@#;,.__.+@#@+,-^^.++@#          @@++
                   ;*  :*@@@##@@@@;++r._j^.+@##@+,.__,,@@++.
                  /*    ........+++++++++++++#@@@@@###@@#++++,
                ,:       ...,@@@#++===----==@@@####,,....+++++
               .:       ......@@##@\   ;   :@####@,,...... +++.
               ;       .........@###,   ;  ;xx#@;,,.....   *;+,
               |       ........,*;xxxx--^--=xxx,........   :+#;
               ;         ......,,;xxxxxxxxxxxxx;,.....     *+#
                ;          ......,::xxxx;.     ......       +.   .
                 *;            .........  +###  ....       / ,. /:| ,.
                   .+:             ...  ;##++##, .      ,#. (..v..;*./
                      **                ##  ###*    .:*&&&+. \.,....<,
                       #&+**==-..,,__  ;##  ###  :,*+&&&&&&&v+#&,,.._/
                      #&&&&*...,::,,.  ##; ,##* .*****;:&&&&&&&&&
                     ,+*+;~*..*** *.* ### ###* *******    *+#&;*
                                      ##,;##    ****    :,  **
             #####    ##   ###  ###,  ########       .#####   ;##  ##
            #######  ;##  #### ,###.  ##########    ########  ### ####
           ###  ###  ### ##########   ####  ####   ,##   ###  #######*
          ### ,###  ##############:   ##     ###  #### ,##   :#### ###  ##;
      ##########    ########### ##   .##    ,###  #######    ##### :######
        ######    .###### ####  ##   ###    ### ######*    :#####   ####
           #############  ####  ################    ######## ###
            #####*  *#*    #:   :###   *###*         *####    #*

```

### PR DESCRIPTION
**Pull request by cironunes**
As you can see, I needed to duplicate a lot of code just to override some properties. It would be nice to fix it in ember-cli.

Closes #19

### CODE REVIEW

File: addon/ng2/utilities/INITIAL_COMMIT_MESSAGE.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Initial Commit from Angular CLI v<%= version %>
```
Comment (cironunes) [inline addon/ng2/utilities/INITIAL_COMMIT_MESSAGE.txt]: I didn't get any original idea to put in here :cry:

File: addon/ng2/commands/new.js [hunk 2]
```diff
@@ -11,7 +20,68 @@ module.exports = NewCommand.extend({
...
-    ]
+    ],
+    run: function(commandOptions, rawArgs) {
+        var packageName = rawArgs[0],
+          message;
+
+        commandOptions.name = rawArgs.shift();
+
+        if (!packageName) {
+            message = chalk.yellow('The `ember ' + this.name + '` command requires a ' +
```
Comment (rodyhaddad) [inline addon/ng2/commands/new.js]: Let's change `ember` to `ng` so we don't forget it later.

File: addon/ng2/commands/new.js [hunk 3]
```diff
@@ -11,7 +20,68 @@ module.exports = NewCommand.extend({
...
+
+            return Promise.reject(new SilentError(message));
+        }
+
+        if (commandOptions.dryRun){
+            commandOptions.skipGit = true;
+        }
+
+        if (packageName === '.') {
+            message = 'Trying to generate an application structure in this directory? Use `ember init` instead.';
```
Comment (rodyhaddad) [inline addon/ng2/commands/new.js]: Same as above

File: addon/ng2/tasks/git-init.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+var Promise  = require('ember-cli/lib/ext/promise');
+var exec     = Promise.denodeify(require('child_process').exec);
+var path     = require('path');
+var pkg      = require('../package.json');
+var fs       = require('fs');
+var template = require('lodash/string/template');
+
+var gitEnvironmentVariables = {
+  GIT_AUTHOR_NAME: 'angular-cli',
+  GIT_AUTHOR_EMAIL: 'angular-cli@angular339.com',
```
Comment (rodyhaddad) [inline addon/ng2/tasks/git-init.js]: @IgorMinar Have any thoughts on which email we should put for the initial commit message of angular-cli projects?

Does Angular have a general contact email?

**Comment by rodyhaddad** [comment]
The remaining issues with the PR are nice to haves, so I'll merge this and open new issues to keep track of them.

File: addon/ng2/utilities/INITIAL_COMMIT_MESSAGE.txt (hunk 1 above)
Comment (rodyhaddad) [inline addon/ng2/utilities/INITIAL_COMMIT_MESSAGE.txt]: Tracked as #23

File: addon/ng2/tasks/git-init.js (hunk 4 above)
Comment (rodyhaddad) [inline addon/ng2/tasks/git-init.js]: Tracked as #24

**Pull request closed without merging**